### ISSUE TITLE
cannot use auto-complete

### ISSUE THREAD

**Issue by timfeirg** [dbcli/mycli#296]
This occured only recently, but I can't pinpoint the version, maybe it's one of the upgraded dependencies that went wrong.

## reproduce

I opened up mycli, type "show ", and this error message comes up right after I typed space:

```python
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/site-packages/prompt_toolkit/interface.py", line 835, in run
    completions = list(buffer.completer.get_completions(document, complete_event))
  File "/usr/local/lib/python2.7/site-packages/mycli/sqlcompleter.py", line 255, in get_completions
    suggestions = suggest_type(document.text, document.text_before_cursor)
  File "/usr/local/lib/python2.7/site-packages/mycli/packages/completion_engine.py", line 85, in suggest_type
    full_text, identifier)
  File "/usr/local/lib/python2.7/site-packages/mycli/packages/completion_engine.py", line 134, in suggest_based_on_last_token
    token_v = token.value.lower()
AttributeError: 'tuple' object has no attribute 'value'
```

## environment

```bash
$ mycli --version
Version: 1.7.1

In [2]: prompt_toolkit.__version__
Out[2]: '1.0.3'
```

**Comment by amjith** [comment]
Duplicate #294. 

Fixed in #295. 

You can try the fix by installing from master as follows:

```
pip install -U https://github.com/dbcli/mycli/archive/master.zip
```

**Comment by lik0914** [comment]
@amjith This is probably the reason why
